lib/rsk-utils.js

```diff
@@ -502,7 +506,25 @@ const getFedsPubKeys = async (bridge) => {
     FEDS_PUBKEYS_LIST.push(removePrefix0x(fedPubKey));
   }
   return FEDS_PUBKEYS_LIST;
-}
+};
+
+/**
+ * Calls `Bridge::voteFeePerKbChange` to set a new fee per kb.
+ * @param {RskTransactionHelper} rskTxHelper 
+ * @param {number} feePerKbInSatoshis the value to be set as the new fee per kb
+ */
+const setFeePerKb = async (rskTxHelper, feePerKbInSatoshis) => {
+
+  await rskTxHelper.getClient().eth.personal.importRawKey(FEE_PER_KB_CHANGE_PK, '');
+  await rskTxHelper.getClient().eth.personal.unlockAccount(FEE_PER_KB_CHANGE_ADDR, '');
+  const bridge = getBridge(rskTxHelper.getClient());
+
+  await sendTransaction(rskTxHelper, bridge.methods.voteFeePerKbChange(feePerKbInSatoshis), FEE_PER_KB_CHANGE_ADDR);
+
+  const finalFeePerKb = await bridge.methods.getFeePerKb().call();
+  expect(Number(finalFeePerKb)).to.equal(Number(feePerKbInSatoshis));
```

Comment:
You could also do a `call` before sending the transaction, and assert the response code

Reply:
A `call` to `voteFeePerKbChange` or `getFeePerKb`?

Reply:
Updated, by calling `sendTxWithCheck`.